### PR TITLE
enable build timestamps

### PR DESCRIPTION
**Pull request by Rozzii**
This commit enables build log timestamps in all pipeline scripts with a 1 second resolution.

### CODE REVIEW

File: jenkins/jobs/bmo_e2e_tests.pipeline [hunk 1]
```diff
@@ -56,9 +58,11 @@ pipeline {
                       currentBuild.result = "FAILURE"
                   }
                 }
-                archiveArtifacts "artifacts*.tar.gz"
-                /* Clean up */
-                sh "make clean-e2e"
+                timestamps {
+                  archiveArtifacts "artifacts*.tar.gz"
```
Comment (tuminoid) [inline jenkins/jobs/bmo_e2e_tests.pipeline:62]: Do we want archiveArtifacts to be in here? Elsewhere its outside timestamps.

**Review by Sunnatillo — COMMENTED** [review]
I checked the logs. This is amazing. Thank you @Rozzii. 
LGTM

File: jenkins/jobs/bmo_e2e_tests.pipeline (hunk 1 above)
Comment (Rozzii) [reply jenkins/jobs/bmo_e2e_tests.pipeline]: no not really I think we are fine without it

**Review by tuminoid — COMMENTED** [review]
/approve

Thanks, this will help a lot in debugging.

**Pull request merged**